tests/unit/mixins/host-app-name-test.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import hostAppName from 'ember-osf/mixins/host-app-name';
+import { module, test } from 'qunit';
+
+module('Unit | Mixin | host app name');
+
+// Replace this with your real tests.
+test('it works', function(assert) {
```

Comment:
At present this looks like just the placeholder code; it doesn't seem to test any of the expected functionality. 

Could I encourage either removing the file, or improving it to make sure that new functionality has test coverage?